### DISCUSSION TITLE
Redirect to wordpress post permanently from vercel

### DISCUSSION THREAD

**Discussion by finemflame** [colbyfayock/next-wordpress-starter#480] [General]
Hii
how can i make a redirect like this subdomain.vercel.app/post-name to wordpressdomain.com/post-name permanently.

**Comment by colbyfayock** [comment]
hey @finemflame there are options with plugins to do this but when using WordPress headlessly, you would need to figure out a way for those redirect configurations to map to your hosting provider

to redirect on Vercel, check out the docs here: https://vercel.com/docs/concepts/edge-network/redirects
